app/services/candidate_interface/section_policy.rb

```diff
@@ -26,7 +26,15 @@ def self.editable_sections
     end
 
     def can_edit?
-      any_offer_accepted? || all_applications_unsubmitted? || editable_section?
+      five_days_after_first_submission? ||
+        any_offer_accepted? ||
+        all_applications_unsubmitted? ||
+        editable_section?
+    end
+
+    def five_days_after_first_submission?
+      @current_application.submitted_at.present? &&
```

Comment:
Did we consider backfilling?

Reply:
The column submitted_at is an existing column :). I am just checking because this check is run by unsubmitted and submitted apps. No need backfilling.

Reply:
oh I see